src/components/application_manager/test/commands/command_request_impl_test.cc

```diff
@@ -156,12 +156,12 @@ TEST_F(CommandRequestImplTest, OnTimeOut_StateAwaitingHMIResponse_SUCCESS) {
 
   CommandPtr command = CreateCommand<UCommandRequestImpl>(msg);
 
-  // Should be called twice:
-  // First -- on `onTimeOut` method call
-  // Second -- on destruction;
-  EXPECT_CALL(event_dispatcher_, remove_observer(_)).Times(2);
-  EXPECT_CALL(app_mngr_,
-              ManageMobileCommand(_, Command::CommandOrigin::ORIGIN_SDL));
+  MessageSharedPtr dummy_msg(CreateMessage());
+  EXPECT_CALL(*am::MockMessageHelper::message_helper_mock(),
+              CreateNegativeResponse(_, _, _, _)).WillOnce(Return(dummy_msg));
```

Comment:
@okozlovlux, I could, but in this case we will get a confuse, where the true nature of static variable of `MockMessageHelper` will be hidden, which may disorientate anyone, who will be dealing with this code in the future.